frame/staking/src/lib.rs

```diff
@@ -1835,3 +2309,56 @@ impl<T, Reporter, Offender, R, O> ReportOffence<Reporter, Offender, O>
 		}
 	}
 }
+
+#[allow(deprecated)]
+impl<T: Trait> frame_support::unsigned::ValidateUnsigned for Module<T> {
+	type Call = Call<T>;
+	fn validate_unsigned(call: &Self::Call) -> TransactionValidity {
+		if let Call::submit_election_solution_unsigned(
+			winners,
+			compact,
+			validator_index,
+			signature,
+		) = call {
+			if let Some(queued_elected) = Self::queued_elected() {
+				if queued_elected.compute == ElectionCompute::Authority {
```

Comment:
This means that the first authority to take the slot wins?

Reply:
This is removed.